src/layout/MainContent.tsx

```diff
@@ -35,7 +35,7 @@ const ResultSection = styled.section`
 const CartContainer = styled.aside`
   background: white;
   padding: 13vh 2.5vw 3vw 3vw;
-  overflow: auto;
```

Comment:
Necessary because otherwise the image will not be shown 😅

Reply:
du coup il y a plus de scroll et on voit plus le bas de la pile ni le bouton démarrer quand il y a trop d'items, c'est bloquant :/ 
J'ai changé le scroll sur ma PR pour les boutons, tu pourrais regarder si ça peut marcher avec

Reply:
Ok, je peux mettre le scroll auto que sur l'axe Y.